build/buildBuildImages.sh

```diff
@@ -117,13 +126,20 @@ function buildGitHubRunnersBaseImage() {
 
 function buildTemporaryFilesImage() {
 	buildGitHubRunnersBaseImage
-	
+	buildGitHubRunnersUbuntuBaseImage
+
 	# Create the following image so that it's contents can be copied to the rest of the images below
 	echo
 	echo "-------------Creating temporary files image-------------------"
 	docker build -t support-files-image-for-build \
 		-f "$BUILD_IMAGES_SUPPORT_FILES_DOCKERFILE" \
 		.
+
+	echo
+	echo "------Creating temporary files image for Ubuntu Focal-----------"
```

Comment:
This is actually not required since the temporary files image is just used to park some files in an image so that it can be used in later stages.

Reply:
agree, but lets keep it for now :) we can always remove it ... we may need it again in case we need to publish ubuntu version of other build images (unlikely but possible)